styx-client/src/main/java/com/spotify/styx/client/Constant.java

```diff
@@ -0,0 +1,32 @@
+/*-
+ * -\-\-
+ * Spotify Styx API Client
+ * --
+ * Copyright (C) 2020 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.styx.client;
+
+import io.grpc.Context;
+
+public final class Constant {
```

Comment:
This name is not very descriptive. What can of constant it contains? is the question that came up to my mind the moment that I read the name of the class. Here are some suggestions: `GrpcAuthConstant` would be more specific or `GrpcAuthUtil` we we are going to expand this further

Reply:
This is not meant to be grpc only constant. It's just for now we don't have anything else. If this feels unclear, could it be `GrpcContextKey`?

Reply:
That sounds good